src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
+    private final double katetA;
+    private final double katetB;
+
+    public RightTriangle(double katetA, double katetB) {
+        this.katetA = katetA;
+        this.katetB = katetB;
+        setColor(new ColorSupplier().getRandomColor());
+    }
+
+    @Override
+    public void toDraw() {
+        System.out.println("Figure: right triangle, area: " + obtainTheArea()
+                + " sq. units, katet A: " + katetA + " units, katet B: " + katetB
+                + " units, color: " + getColor());
+    }
+
+    @Override
+    public double obtainTheArea() {
+        return (katetA * katetB) / 2;
```

Comment:
The method name `obtainTheArea` is not following the naming convention for methods. It's better to use a simple and direct name like `getArea`.